lib/3scale/backend/async_redis/client.rb

```diff
@@ -16,6 +16,19 @@ class << self
           # @param opts [Hash] Redis connection options
           # @return [Async::Redis::Client]
           def call(opts)
+            return connect_tcp(opts) if url_present?(opts[:url])
+
+            connect_unix(opts)
```

Comment:
In this case do we support `unix://` URLs and how?

Reply:
`unix://` URLs are supported and inf fact they are the only way to setup a unix path on apisonator. So it matches how the porta side works

Reply:
But then URL will be present and `connect_tcp` will be used instead of `connect_unix`? Maybe I'm missing something obvious in the logic..

Reply:
The `:url` param is removed here:

https://github.com/3scale/apisonator/blob/e79bef023dc250e7a916a4afe93d6fa0c34599e6/lib/3scale/backend/storage_helpers.rb#L280-L282

This is because `redis-rb` expects `:url` for TCP and `:path `for unix sockets, so we must emulate the same behavior for `async-redis`